modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -563,6 +572,23 @@ public void keyRemoved(
 				});
 		_graphQLRequestContextValidators = ServiceTrackerListFactory.open(
 			bundleContext, GraphQLRequestContextValidator.class);
+
```

Comment:
@rcsdnj use the `ServiceTrackerListFactory.open()` instead to make the service tracker open a lazy operation.

Reply:
Done, could you check please?